Blogplace.Web/Exceptions/CustomException.cs

```diff
@@ -4,11 +4,11 @@ namespace Blogplace.Web.Exceptions;
 
 public abstract class CustomException : Exception
 {
-    public CustomException(string? message) : base(message)
```

Comment:
lepiej zostawić public, żeby do nadpisywania podpowiadało public

tutaj protected i tak nic nie da bo to klasa abstrakcyjna, i tak nikt tego nie stworzy